### ISSUE TITLE
`get_visible_line_count()` in RichTextLabel is wrong if text contains newlines

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot#59359]
### Godot version

3.5 beta2

### System information

Windows 10 x64

### Issue description

When text contains newlines, `get_visible_line_count()` is off by the number of newlines. I use this code to automatically scroll the RTL:
```GDScript
func _process(delta: float) -> void:
	if visible_characters < text.length():
		visible_characters += 1
	
	if get_visible_line_count() > 6:
		get_v_scroll().value += 3
```
To the left is text with single, autowrapped line. To the right is text with one newline. The visible line count starts from 0 and is 1 too low.
![godot_IFcfZc4Tvu](https://user-images.githubusercontent.com/2223172/159176651-8abb681f-cb63-4270-a761-5b62d7fdfbce.gif)

Does not happen on master.

### Steps to reproduce

1. Add RichTextLabel
2. Write some text that wraps
3. Animate `visible_characters`
4. Check the results of `get_visible_line_count()`
5. Do the same with text with line breaks

### Minimal reproduction project

[ReproductionProject.zip](https://github.com/godotengine/godot/files/8311913/ReproductionProject.zip)

**Comment by AtlaStar** [comment]
`RichTextLabel` also reports a `percent_visible` greater than 100% on completion, and `get_total_character_count` is reporting that the second `RichTextLabel` has 1 less character. I performed a diff of the text in both to confirm that the second is 100% identical to the first outside of a space being replaced with a newline in the second.

**Comment by akien-mga** [comment]
Fixed by #59765.
